src/main/java/com/amazonaws/services/simpleworkflow/flow/WorkflowContext.java

```diff
@@ -93,18 +103,26 @@
      * executed the second call to isVersion returns <code>false</code> even if
      * it is executed for the first time.
      * <p>
-     * 
+     *
      * <pre>
+     <<<<<<< HEAD
      * if (workflowContext.isImplementationVersion(&quot;comp1&quot;, 1)) {
+     =======
+     * if (workflowContext.isVersion(&quot;comp1&quot;, 1)) {
+     >>>>>>> d7ef09c... save for test use.
```

Comment:
Probably during conflict resolution you forgot to fix this. Please fix it.

Reply:
Fixed.